app/views/incomes/YourIncomeCalculationView.scala.html

```diff
@@ -69,7 +71,7 @@
                                 <p class="govuk-body">@Messages("tai.income.calculation.heading.employerInfo", model.employerName)</p>
                             }
 
-                            <p class="govuk-body">@Messages("tai.income.calculation.potentially.ceased.lede")</p>
+                            <p class="govuk-body">@Messages("tai.income.calculation.potentially.ceased.lede") </p>
```

Comment:
Does this single white space inside the p tag matter?

Reply:
oh looks like intelliJ  does some weird things  as that line was untouched :)
nothing to be concerned about

Reply:
okay nice 👍